src/main/java/de/upb/crypto/clarc/incentive/CreditInstance.java

```diff
@@ -0,0 +1,59 @@
+package de.upb.crypto.clarc.incentive;
+
+import de.upb.crypto.clarc.protocols.arguments.SigmaProtocol;
+import de.upb.crypto.clarc.protocols.parameters.Announcement;
+import de.upb.crypto.clarc.protocols.parameters.Challenge;
+import de.upb.crypto.clarc.protocols.parameters.Response;
+import de.upb.crypto.craco.sig.ps.PSExtendedVerificationKey;
+import de.upb.crypto.craco.sig.ps.PSSignature;
+import de.upb.crypto.craco.sig.ps.PSSigningKey;
+import de.upb.crypto.math.interfaces.structures.GroupElement;
+import de.upb.crypto.math.structures.zn.Zp;
+
+public class CreditInstance {
+	IncentiveSystemPublicParameters pp;
```

Comment:
visibility of variables: private, public, protected?

Reply:
I used package-private so I can drop the getter. Thought it would be ok for the prototype.